docs/GraFx-Fonts/guides/upload-fonts/index.md

```diff
@@ -0,0 +1,71 @@
+# Upload fonts
+
+## License
+
+Adding fonts to CHILI GraFx requires a valid license. Before uploading, check your font supplies (foundry) if you have a valid license to use your font's for your application.
```

Comment:
shouldn't this be "font supplier" instead of "supplies", don't know the technical details here, so might be wrong myself

Reply:
Totally right